docs/startup-order.md

```diff
@@ -75,7 +74,7 @@ script:
         exec $cmd
 
     You can use this as a wrapper script as in the previous example, by setting
-    `entrypoint: ./wait-for-postgres.sh db`.
+    `command: ["./wait-for-postgres.sh", "db", "python", "app.py"]`.
```

Comment:
so - what's the difference between using `entrypoint` and `command` in these examples - its a bit weird inthe context of the change in `compose-file.md`